modules/fluid_properties/doc/content/source/userobjects/TabulatedFluidProperties.md

```diff
@@ -94,10 +111,11 @@ divided into 50 and 100 equal points, respectively, then the input file syntax n
       pressure_max = 10e6
       num_T = 50
       num_p = 100
-    [../]
+    []
   []
 []
 ```
+Note that Bi___ should be replaced with Bicubic or Bilinear depending on desired the Interpolation method
```

Comment:
- Blank line needed after code block
- Quotes around "Bi___", "Bicubic", and "Bilinear"
- on the desired
- lowercase "interpolation"
- Period